src/services/netflix/NetflixApi.ts

```diff
@@ -188,7 +188,7 @@ class _NetflixApi extends ServiceApi {
 				});
 				this.session = this.extractSession(responseText);
 			}
-			if (this.session && this.session.profileName) {
+			if (this.session?.profileName != null) {
```

Comment:
I don't think we need to compare against `null` here. Just `this.session?.profileName` should work, as it would only be `true` if `profileName` is a non-empty string.

Reply:
I'm not sure, whether it is possible to have a profile with an empty string as a name. I chose to make the not nullish comparison, because I'm pretty sure when you are not logged in, `profileName` is `null`, and when the session is not set, it is either `null` or `undefined`, which are both covered by this comparison.

But of course if you prefer, I can change it to a truthy comparison.

Reply:
> I'm not sure, whether it is possible to have a profile with an empty string as a name.

I don't think so, it should be safe to change it.

Reply:
Actually, it might be the case that we don't have access to the profile name... Hm, maybe we can leave it like this then.